config/manager/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: status
+  namespace: system
+spec:
+  type: NodePort
```

Comment:
Service should be ClusterIP and .spec.clusterIP should be set to none: https://kubernetes.io/docs/concepts/services-networking/service/#headless-services

Reply:
Done